spec/downloader/base_spec.rb

```diff
@@ -135,6 +135,40 @@
     end
   end
 
+  context 'with download_retry_count' do
+    before do
+      stub_request(:get, uri).to_return(status: 503)
+    end
+    let(:instance) { described_class.new(uploader) }
+    subject { instance.download uri, download_retry_count }
+    context 'when failed' do
+      context 'when download_retry_count == 0 ' do
+        let(:download_retry_count) { 0 }
+        it { expect { subject }.to raise_error CarrierWave::DownloadError }
+
+        it do
```

Comment:
Don't omit the example name (`it '...'`), unless it can be read naturally using the one-liner syntax.
https://relishapp.com/rspec/rspec-core/docs/subject/one-liner-syntax